docs/sdk-and-tools/rest-api/transactions.md

```diff
@@ -479,3 +521,333 @@ Transaction status retrieved successfully.
+                        "sender": "erd1...",
+                        "value": "10000000000000000000"
+                    }
+                }
+            ],
+            "smartContractResults": [
+                {
+                    "txFields": {
+                        "gasLimit": 10,
+                        "gasPrice": 1000,
+                        "receiver": "erd1...",
+                        "sender": "erd1...",
+                        "value": "10000000000000000000"
+                    }
+                }
+            ],
+            "rewards": [
+                {
+                    "txFields": {
+                        "gasLimit": 10,
+                        "gasPrice": 1000,
+                        "receiver": "erd1...",
+                        "sender": "erd1...",
+                        "value": "10000000000000000000"
+                    }
+                }
+            ]
+        }
+    },
+    "error": "",
+    "code": "successful"
+}
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## <span class="badge badge-primary">GET</span> **Get Transactions Pool for a Sender**
+
+`https://gateway.elrond.com/transaction/pool?by-sender=:sender:`
+
+This endpoint allows one to fetch all the transactions of a sender from the transactions pool.
+
+### Default
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Request-->
+
+Query Parameters
+
+| Param     | Required                                  | Type     | Description                          |
+|-----------|-------------------------------------------|----------|--------------------------------------|
+| by-sender | <span class="text-normal">REQUIRED</span> | `string` | The Address of the sender.           |
+
+Example:
+
+`https://gateway.elrond.com/transaction/pool?by-sender=erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th`
+
+<!--Response-->
+
+🟢 200: OK
+
+Transaction status retrieved successfully.
+
+```
+{
+  "data": {
+    "txPool": {
+      "transactions": [
+        {
+          "txFields": {
+            "hash": "1daea5..."
+          }
+        }
+      ]
+    }
+  },
+  "error": "",
+  "code": "successful"
+}
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+### Using custom fields
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Request-->
+
+Query Parameters
+
+| Param     | Required                                  | Type     | Description                          |
+|-----------|-------------------------------------------|----------|--------------------------------------|
+| by-sender | <span class="text-normal">REQUIRED</span> | `string` | The Address of the sender.           |
+| fields    | <span class="text-normal">OPTIONAL</span> | `string` | A list of the fields to be included. |
+
+As seen above, if the `fields` item is empty, only the transaction hash will be displayed.
+
+Example request with fields:
+
+`https://gateway.elrond.com/transaction/pool?by-sender=erd1at9...&fields=sender,receiver,value`
+
+All possible values for fields item are:
+
+- hash           
```

Comment:
same note here, hash is default and nothing happens if it is added to fields

Reply:
would leave it as it is